policyengine_us/variables/household/expense/childcare/childcare_expenses.py

```diff
@@ -13,9 +13,13 @@ def formula(spm_unit, period, parameters):
             spm_unit, period, ["pre_subsidy_childcare_expenses"]
         )
         # States where we model childcare subsidies.
-        STATES_WITH_CHILD_CARE_SUBSIDIES = ["CA", "CO"]
+        STATES_WITH_CHILD_CARE_SUBSIDIES = ["CA", "CO", "NE"]
         subsidy_variables = [
-            i.lower() + "_child_care_subsidies"
+            (
+                i.lower() + "_child_care_subsidy"
+                if i == "NE"
+                else i.lower() + "_child_care_subsidies"
```

Comment:
might want to consolidate this by creating a general `ne_child_care_subsidies.py` - not sure if there are other subsidy programs